source/endpoint-reference/endpoint-tabbook.rst

```diff
@@ -652,37 +652,47 @@ POST body parameters
-------- ------ -------- ----------------------------- ----------------------------------------------------------------------------------------------------------------
-weight  url    null     Provide a weight for the
-                        tabbook generation, if the
-                        weight is omitted from the
-                        request, the currently
-                        selected weight is used. If
-                        "null" is provided, then the
-                        tabbook generation will be
-                        unweighted.
-======= ====== ======== ============================= ================================================================================================================
+========== ====== ======== ============================= ================================================================================================================
+Name       Type   Default  Description                   Example
+========== ====== ======== ============================= ================================================================================================================
+filter     object null     Filter by Crunch Expression.  .. code:: json
+                           Variables used in the filter
+                           should be fully-expressed
+                           urls.                          [{"filter":
+                                                          "https://app.crunch.io/api/datasets/45fc0d5ca0a945dab7d05444efa3310a/filters/5f14133582f34b8b85b408830f4b4a9b/"
+                                                          }]
+---------- ------ -------- ----------------------------- ----------------------------------------------------------------------------------------------------------------
+where      object null     Crunch Expression signifying  .. code:: json
+                           which variables to use
+                                                          {"function": "select",
+                                                           "args": [{"map": {
+                                                            "https://app.crunch.io/api/datasets/45fc0d5ca0a945dab7d05444efa3310a/variables/000004/": {
+                                                             "variable": "https://app.crunch.io/api/datasets/45fc0d5ca0a945dab7d05444efa3310a/variables/000004/"},
+                                                            "https://app.crunch.io/api/datasets/45fc0d5ca0a945dab7d05444efa3310a/variables/000003/": {
+                                                             "variable": "https://app.crunch.io/api/datasets/45fc0d5ca0a945dab7d05444efa3310a/variables/000003/"}}}]}
+---------- ------ -------- ----------------------------- ----------------------------------------------------------------------------------------------------------------
+variables  array  null     list of variables or folder   .. code:: json
```

Comment:
- Capitalize: `list of variables or folder urls to include.`

- Replace: `on this list` with `in this list`.

- Replace: `on its subfolders` with `in its subfolders`.

- Add a period to the send of this sentence: `The folders included on this list will include all the variables on its subfolders`

Reply:
Done, thanks!